src/views/Login/login.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import loginAction from "../../redux/actions/loginAction";
+import Button from "../Button";
+import LoginForm from "./LoginForm";
+import SocialAuth from "./socialAuth";
+
+class LoginPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { email: "", password: "" };
```

Comment:
The constructor is now irrelevant as the bind statement has been removed. Refactor to remain only with the state.

Reply:
@kimotho-njoki I have removed the constructor method.